python/ray/tune/README.rst

```diff
@@ -1,7 +1,7 @@
-Ray.tune: Efficient distributed hyperparameter search
-=====================================================
+Ray.tune: Hyperparameter Optimization Framework
```

Comment:
I felt like there was an overload of adjectives on the docs page, so cleaned this up a bit.

Reply:
Suggestions on description welcome...

Reply:
I think it's good; it definitely declutters the sidebar, and we can work on the teasers and stuff once we have closer integration with the autoscaler and such.